src/System.Text.Utf8String/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTrademark("")]
+
+[assembly: InternalsVisibleTo("System.Text.Utf8String.Tests,PublicKey="+
```

Comment:
Making these classes public is probably not the right thing to do right now. This entire assembly is going to be re-designed and possibly removed at some point in the near future. So, I'm leaving it as is for now.